datashader/tests/test_pandas.py

```diff
@@ -652,7 +708,7 @@ def test_categorical_sum_binning(df):
 
 
 @pytest.mark.parametrize('df', dfs)
-def test_categorical_max(df):
+def test_categorical_max2(df):
```

Comment:
Keeping the existing categorical max test but renaming it so it doesn't overwrite the new one which is above it in this file.